From: Eric Biggers <ebiggers@google.com>
Date: Mon, 20 Aug 2018 16:37:37 -0700
Subject: libe2p: fix verity flag bit

The verity flag was mapped to EXT4_PROJINHERIT_FL, presumably due to a
copy+paste error.  Fix it.

Fixes: faae7aa00df0 ("Reserve codepoints for the fsverity feature.")
Signed-off-by: Eric Biggers <ebiggers@google.com>
Signed-off-by: Theodore Ts'o <tytso@mit.edu>
Origin: Upstream, https://git.kernel.org/pub/scm/fs/ext2/e2fsprogs.git/commit/?h=maint&id=7e5a95e3d597
---
 lib/e2p/pf.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/e2p/pf.c b/lib/e2p/pf.c
index 1c4bdde..884f167 100644
--- a/lib/e2p/pf.c
+++ b/lib/e2p/pf.c
@@ -46,7 +46,7 @@ static struct flags_name flags_array[] = {
 	{ FS_NOCOW_FL, "C", "No_COW" },
 	{ EXT4_INLINE_DATA_FL, "N", "Inline_Data" },
 	{ EXT4_PROJINHERIT_FL, "P", "Project_Hierarchy" },
-	{ EXT4_PROJINHERIT_FL, "V", "Verity" },
+	{ EXT4_VERITY_FL, "V", "Verity" },
 	{ 0, NULL, NULL }
 };
 
